api/src/main/java/marquez/db/OpenLineageDao.java

```diff
@@ -773,7 +820,7 @@ default String formatNamespaceName(String namespace) {
   }
 
   default JobType getJobType(Job job) {
-    return JobType.BATCH;
+    return Utils.isStreamingJob(job) ? JobType.STREAM : JobType.BATCH;
```

Comment:
_Minor_: We can go one step further and define this method within the context of [`LineageEvent.Job`](https://github.com/MarquezProject/marquez/blob/main/api/src/main/java/marquez/service/models/LineageEvent.java#L204) class:

```bash
# Returns the Marquez job type.
job.type() 
```